api/osf_api.py

```diff
@@ -0,0 +1,6 @@
+from pythosf import client
+
+def create_project(session, title='osf selenium test'):
+    node = client.Node(session=session)
+    node.create(title=title)
```

Comment:
The next version will likely return the object from the create method, so this should be much simpler in the future. Thank you for the feedback and let me know if you happen to find anything else like that.